dashboard/src/components/InstalledPackages/InstalledPackagesHeader.tsx

```diff
@@ -32,7 +32,7 @@ export default function InstalledPackagesHeader({
 
         <div className="w-1/3">
           <input
-            className="border border-[#ced4da] rounded p-1 text-sm w-11/12"
+            className="border-border-gray  rounded p-1 text-sm w-11/12"
```

Comment:
pay attention you are using `grey` in ClusterList, and here your using `gray`
choose one